lightkurve/correctors/designmatrix.py

```diff
@@ -269,6 +269,7 @@ def append_constant(self, prior_mu=0, prior_sigma=np.inf, inplace=False):
             dm = self.copy()
         extra_df = pd.DataFrame(np.atleast_2d(np.ones(self.shape[0])).T, columns=['offset'])
         dm.df = pd.concat([self.df, extra_df], axis=1)
+        dm.columns = list(dm.df.columns)
```

Comment:
(This is a small bug in DesignMatrix; `dm.columns` was not updated after calling `dm.append_constant()`)